src/charm.py

```diff
@@ -179,6 +185,18 @@ def _pebble_layer(self) -> pebble.LayerDict:
                     ),
                     "startup": "enabled",
                 },
+                "prometheus-exporter": {
+                    "override": "replace",
+                    "summary": "prometheus exporter",
+                    "command": "/usr/bin/python3 /usr/bin/prometheus_exporter.py",
+                    "startup": "enabled",
+                    "environment": {
+                        "WAZUH_API_HOST": "localhost",
+                        "WAZUH_API_PORT": "55000",
+                        "WAZUH_API_USERNAME": "wazuh",
+                        "WAZUH_API_PASSWORD": self.state.api_credentials["wazuh"],
```

Comment:
I'd rather use a dedicated user for this, with read-only permissions, since we're using an external prometheus exporter, that would limit a bit the risk